tests/test_app_builder/test_with_gov.rs

```diff
@@ -34,3 +34,171 @@ fn building_app_with_custom_gov_should_work() {
         .to_string()
     );
 }
+
+#[test]
+fn building_app_with_default_gov_vote_should_work() {
```

Comment:
Just to consider. Reading the name of test "should_work" and seeing `unwrap_err` later in test seems counterintuitive

```suggestion
fn default_failing_gov_vote() {
```